librz/util/buf.c

```diff
@@ -381,7 +468,16 @@ RZ_API RzBuffer *rz_buf_new_with_bytes(RZ_NULLABLE const ut8 *bytes, ut64 len) {
 
 // copied from librz/io/cache.c:rz_io_cache_read
 // ret # of bytes copied
-RZ_API RzBuffer *rz_buf_new_with_io(RZ_NONNULL void *iob, int fd) {
+
+/**
+ * \brief Creates a new buffer with from an RzIOBind and a fd.
+ * \param iob ...
+ * \param fd ...
+ * \return Return the new allocated buffer.
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_IO back end.
```

Comment:
The function creates a new buffer wrapping access to a file descriptor through the RzIOBind methods specified in `iob`.


Or something like that...